crates/indexer/src/models/transactions.rs

```diff
@@ -219,6 +220,33 @@ impl Transaction {
             APITransaction::PendingTransaction(..) => {
                 unreachable!()
             },
+            APITransaction::SystemTransaction(sys_txn) => {
+                let (wsc, wsc_detail) = WriteSetChangeModel::from_write_set_changes(
+                    &sys_txn.info.changes,
+                    sys_txn.info.version.0 as i64,
+                    block_height,
+                );
+                (
+                    Self::from_transaction_info(
+                        &sys_txn.info,
+                        None,
+                        transaction.type_str().to_string(),
+                        0,
+                        block_height,
+                        epoch,
+                    ),
+                    Some(TransactionDetail::System(
+                        SystemTransaction::from_transaction(sys_txn, block_height),
+                    )),
+                    EventModel::from_events(
+                        &sys_txn.events,
+                        sys_txn.info.version.0 as i64,
+                        block_height,
+                    ),
+                    wsc,
+                    wsc_detail,
+                )
```

Comment:
we don't need to add anything here. Just add a default that returns null is sufficient. I can handle this.